integrations/pinecone/tests/test_document_store.py

```diff
@@ -87,6 +87,9 @@ def test_write_documents(self, document_store: PineconeDocumentStore):
         docs = [Document(id="1")]
         assert document_store.write_documents(docs) == 1
 
+    @pytest.mark.skip(reason="Pinecone supports overwriting by default, but it takes a while for it to take effect")
```

Comment:
Let's mark this as `xfail` instead so that we can track when it starts to work again.